model_extension/1comp/template/in.1comp.template

```diff
@@ -39,8 +39,8 @@ variable SIGMAHC 	equal 	v_SIGMAHCRAW/v_VLPDiameterRAW
 variable VLPLinkerLJcutoff	equal v_SIGMAHC*(2^(1.0/6.0))	# 2^(1/6) times the VLP-linker interaction lengthscale
 variable DELTA		equal 	(1+v_LinkerDiameter)*0.5-v_SIGMAHC	# in reduced units
 variable KAPPA		equal 	v_VLPDiameterRAW*sqrt(8*v_lB*v_SALTCONC*PI*0.6022141)	# in reduced units of (1 / vlp diameter)	
-variable VLPChargeDLVO	equal	(v_VLPCharge*exp(v_KAPPA*0.5)/(1+v_KAPPA*0.5))*(1.000676522)	# KAPPA is already in reduced units of the simulation
-variable LinkerChargeDLVO equal (v_LinkerCharge*exp(v_KAPPA*v_LinkerDiameter*0.5)/(1+v_KAPPA*v_LinkerDiameter*0.5))*(1.000676522) # both KAPPA and Linker diameter are in reduced units
+variable VLPChargeDLVO	equal	(v_VLPCharge*exp(v_KAPPA*0.5)/(1+v_KAPPA*0.5))*(1.00065763)	# KAPPA is already in reduced units of the simulation

```

Comment:
I think this change is not updated at my end. You chose 1e=1.60217*10^-19 Coulombs to get this (1.00065763) factor. The 2 comp script in template still has the old factor (1.000676522).